src/main/scala/edu/ie3/simona/model/control/TransformerControlGroup.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.control
+
+import breeze.math.Complex
+import edu.ie3.powerflow.model.NodeData.StateData
+import edu.ie3.powerflow.model.PowerFlowResult.SuccessFullPowerFlowResult
+import edu.ie3.simona.model.control.TransformerControlGroup.RegulationCriterion
+import tech.units.indriya.ComparableQuantity
+
+import java.util.UUID
+import javax.measure.quantity.Dimensionless
+
+/** Business logic for a transformer control group. It's main purpose is to
+  * determine, if there is any regulation need and if yes, in what circumference
```

Comment:
Circumference? -- suggested replacement with region or area. 
Or does it mean something entirely different -- e.g., type of regulation as suggested in the following parentheses?

Reply:
IMO it should mean something like: to what extent a regulation is needed. I will change it to have a better understanding.